demo/meter/meter.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta http-equiv="X-UA-Compatible" content="ie=edge">
+		<meta charset="UTF-8">
+		<link rel="stylesheet" href="../styles.css" type="text/css">
+		<script src="../../node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+		<script type="module">
+			import '../../components/demo/demo-page.js';
+			import '../../components/meter/meter-linear.js';
+		</script>
+	</head>
+	<body unresolved>
+
+		<d2l-demo-page title="meter">
```

Comment:
Nit: for `title` I've been putting the name of the component.  i.e. `d2l-meter-linear` in this case.

Reply:
oh... i suspect you are planning to put them all on one.  that's ok too.

Reply:
For the other meters will it be better to have them in separate demo files or have multiple `d2l-demo-page`?

Reply:
In the case of buttons, I split them out since there were several demos for each.  I think it's fine to combine them, but if there's more then a few demos for each, then I'd probably split them.